package/CHANGELOG

```diff
@@ -28,7 +28,7 @@ Deprecations
 
 03/06/22 IAlibay, melomcr, mdd31, ianmkenney, richardjgowers, hmacdope,
          orbeckst, scal444, p-j-smith, edisj, Atharva7K, ojeda-e, jbarnoud,
-         yangtcai
+         yangtcai, AnirG
```

Comment:
Version 2.1.0 has now been released, you'll need to update your branch and add your changelog entry to 2.2.0 instead.

Reply:
Sorry, didn't realize 2.1.0 was out already. I have revoked the changes and have updated on 2.2.0.